doc/elements.md

```diff
@@ -88,6 +88,10 @@ Attribute | Type | Default | Description
 `atol` | number | 1e-8 | Absolute tolerance for `comparison="relabs"`.
 `digits` | integer | 2 | number of digits that must be correct for `comparison="sigfig"` or `comparison="decdig"`.
 `allow-complex` | boolean | False | Whether or not to allow complex numbers as answers. If the correct answer `ans` is a complex object, you should use `import prairielearn as pl` and `data['correct_answer'][answers-name] = pl.to_json(ans)`.
+`hide-help-text` | boolean | False | Hide question mark at the end of the input displaying required input parameters.
```

Comment:
Naming: maybe `show-help-text` for consistency with other potential boolean attribute names? Not sure if I like naming with affirmative or positive verbs :/ @mwest1066 any thoughts?

Reply:
Again, I think we should keep consistency. `pl-checkbox` has `hide-help-text`. So if we change here, I think we should change there as well.

Reply:
Agreed, though we don't have the best migration/deprecation plan for elements and attributes. I'd be happy to rename the attribute to the "right" thing on `pl-checkbox` and update the docs but support both versions in code indefinitely.